sdk/src/ingredient.rs

```diff
@@ -930,7 +931,7 @@ impl Ingredient {
 
         // retrieve the manifest bytes from embedded, sidecar or remote and convert to store if found
         let (result, manifest_bytes) = match Store::load_jumbf_from_stream(format, stream) {
-            Ok(manifest_bytes) => {
+            Ok((manifest_bytes, _)) => {
```

Comment:
Can be collapsed to Store::from_stream

Reply:
The problem here is that we need the manifest bytes for this call:
```rust
ingredient.update_validation_status(result, manifest_bytes, &validation_log)?;
```